src/Planners/include/BipedalLocomotion/Planners/SO3Planner.h

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file SO3Planner.h
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_PLANNERS_SO3_PLANNER_H
+#define BIPEDAL_LOCOMOTION_PLANNERS_SO3_PLANNER_H
+
+#include <manif/SO3.h>
+
+namespace BipedalLocomotion
+{
+namespace Planners
+{
+
+/**
+ * Representation used in the SO3Planner
+ */
+enum class Representation
```

Comment:
This is a little vague. Consider it would be ``BipedalLocomotion::Planners::Representation`` (of what?).

Reply:
Yes you're right, any suggestion? It would be nice to have `SO3Planner::Representation` (we can change the keyword `representation)` 
Unfortunately, it is not possible because `Representation` is the template argument of `SO3Planner`

Reply:
Adding a namespace ``SO3``?

Reply:
What do you have in mind?

Reply:
> What do you have in mind?

Right now nothing. I think that the namespace `SO3` should be enough 
```cpp
namespace SO3
{
enum class Trivialization
{
   Left, 
   Right,
};
}
```
is it fine with you?

Reply:
Maybe it would be better to not restrict this to SO3 namespace.

And do you think `VelocityTrivialization` is more descriptive than `Trivialization`?

Reply:
> Maybe it would be better to not restrict this to SO3 namespace.
> 
> And do you think `VelocityTrivialization` is more descriptive than `Trivialization`?

Or `LieGroupTrivialization` directly in the namespace `BipedalLocomotion`. The question is? where we should put the file containing the `enum`?

Reply:
For the time being, I guess you can leave it here. In case we will need it somewhere else, we may find another home for it.

Reply:
This should solve the problem https://github.com/dic-iit/bipedal-locomotion-framework/pull/84/commits/4dad9ead34eeedca3898f5303b5dc5f0592ea090